.auth0/rules/rule.js

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Please rename file to `attach-metadata.js`

Reply:
and run prettier

Reply:
done!